examples/generatorOptions.md

```diff
@@ -0,0 +1,127 @@
+# Generator Options
+
+Kustomize provides options to modify the behavior of ConfigMap and Secret generators. These options include
+ 
+ - disabling hash appended to the generated resources
+ - adding labels to generated resources
+ - adding annotations to generated resources
+ - changing shell and arguments for getting data from commands
+ - changing timeout for executing commands
+ 
+This demo shows how to use these options. First create a workspace.
+```
+DEMO_HOME=$(mkdir -d)
+```
+
+Create a kusotmization and add a ConfigMap generator to it.
+
+<!-- @createCMGenerator @test -->
+```
+cat > $DEMO_HOME/kustomization.yaml << EOF
+configMapGenerator:
+- name: my-configmap
+  literals:	
+  - foo=bar
+  - baz=qux
+EOF
+```
+
+## disabling hash
+By default, ConfigMap generator adds a suffix to the names of generated ConfigMaps.
```

Comment:
Show that this is the default, by running build right here, and grepping for the hash in a test following the `test 1 == ...` pattern as in multibases/README.md

...or some other test style  - just so there's test coverage.  Otherwise there's no point to editing the kustomization.yaml file.